actix/src/sec-12-coming-from-akka.md

```diff
@@ -0,0 +1,52 @@
+Actix is an actor model for Rust very similar to how Akka is also an actor model for Scala & Java. This page aims to introduce Actix from the perspective of someone familiar with Akka.
```

Comment:
`...similar to how Akka` -> `similar to Akka which is actor model for Scala & Java`